src/app/traveler/flights/flights.component.ts

```diff
@@ -114,18 +117,19 @@ export class FlightsComponent implements OnInit {
         .subscribe(
           () => {
             this.modalService.dismissAll();
+            this.toastsService.showSuccess('Flight booked. View in My Bookings', 'yay!');
             this.flights = this.flights.filter(
               (flight) => flight !== this.selectedFlight
             );
           },
           (error) => {
             this.modalService.dismissAll();
-            alert('Error booking ticket: Status ' + error.error.status);
+            this.toastsService.showError('Something went wrong booking flight', 'Error');
           }
         );
     } else if (result.error) {
       this.modalService.dismissAll();
-      alert("Error processing payment: Token creation failed.");
+      this.toastsService.showError('Incorrect card information.', 'Error');
```

Comment:
We don't know for sure that the card information is incorrect here. The token creation could have failed for another reason, like a problem with Stripe's servers.